src/Sulu/Bundle/SecurityBundle/EventListener/LogoutEventSubscriber.php

```diff
@@ -39,7 +39,7 @@ public function onLogout(LogoutEvent $logoutEvent): void
         $adminUrl = $this->urlGenerator->generate('sulu_admin');
         $request = $logoutEvent->getRequest();
 
-        if (!\str_starts_with($request->getPathInfo(), $adminUrl)) {
+        if (!str_starts_with($request->getPathInfo(), $adminUrl)) {
```

Comment:
This `\` is expected in our code style:

```suggestion
        if (!\str_starts_with($request->getPathInfo(), $adminUrl)) {
```